hadoop-tools/hadoop-aws/src/main/java/org/apache/hadoop/fs/s3a/DefaultS3ClientFactory.java

```diff
@@ -165,6 +175,8 @@ private <BuilderT extends S3BaseClientBuilder<BuilderT, ClientT>, ClientT> Build
             .pathStyleAccessEnabled(parameters.isPathStyleAccess())
             .build();
 
+    builder.fipsEnabled(parameters.isFipsEnabled());
```

Comment:
we already set this on line 265, so can remove from here.

Reply:
ahh